process-engine/skeleton
# 🐛 add process_engine_server to the list of things to build

## What did you change?

I Added the process_engine_server to the list of things that should be built when running `npm run build` in the skeleton root-folder.

In the past, the process_engine_server consisted only of a single JS file. This has changed, and it now has two classes, an ioc_module and is written in typescript (because we needed decorators). Because of this, the process_engine_server needs building as well.

## How can others test the changes?

- Run `npm run build` in the skeleton-root without this fix
- See how the process_engine_server isn't being built
- Run `npm run build` in the skeleton-root without this fix
- See how the process_engine_server is now being built

## PR-Checklist

Please check the boxes in this list after submitting your PR:
